docs/staging-production-deployment.md

```diff
@@ -0,0 +1,93 @@
+# Staging / Production Deployment
+
+The steps to launch Telescope in staging or production mode are almost identical. The only 2 differences between the 2 deployment types are:
+
+- Some of the variables in the `.env` files need to be manually set with the values `staging` or `production`
+- For the first deployment, Telescope must be launched using the appropriate `docker-compose.yml` file for staging or production.
+
+## Prerequisites
+
+- [Docker and Docker-compose](https://github.com/Seneca-CDOT/telescope/blob/master/docs/environment-setup.md#docker-and-docker-compose-set-up)
+- [Node.js (npm)](https://nodejs.org/en/download/)
+- [pm2](https://pm2.keymetrics.io/docs/usage/pm2-doc-single-page/) (optional)
+
+## Directory Structure
+
+The project needs to be set up using a specific directory structure to make sure that Telescope has access to the SSL certificates, redis data and elasticsearch data, and to prevent the certificates and the stored data from being deleted during a redeployment.
+
+Parent directory<br>
+|<br>
+|\_\_autodeployment<br>
+|<br>
+|\_\_certbot<br>
+|<br>
+|\_\_elastic-data<br>
+|<br>
+|\_\_redis-data<br>
+|<br>
+|\_\_telescope<br>
```

Comment:
I tried following the instructions here but didn't have a domain to replace the `example.org` domain in the Let's Encrypt and Nginx so that was the furthest I've gotten :( 

I do have a suggestion for the folder structure though:
```bash
├── parent-directory
│   ├── autodeployment
│   ├── certbot
│   ├── elastic-data
│   ├── redis-data
│   └── telescope
```
just different formating so nothing huge

Reply:
Yes! that looks better, I'll add the change. Thank you!